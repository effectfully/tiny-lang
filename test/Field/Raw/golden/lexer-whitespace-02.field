for

i
	=
0to
100
do assert T; end



;



T