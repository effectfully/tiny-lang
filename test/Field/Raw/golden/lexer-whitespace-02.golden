EStatement (EFor (Var "i") (EConst (Some 0)) (EConst (Some 100)) [EAssert (EConst (Some (UniConst Bool True)))]) (EConst (Some (UniConst Bool True)))